pkgs/top-level/all-packages.nix

```diff
@@ -13557,9 +13557,10 @@ with pkgs;
 
   mozart2 = callPackage ../development/compilers/mozart {
     emacs = emacs-nox;
-    jre_headless = jre8_headless; # TODO: remove override https://github.com/NixOS/nixpkgs/pull/89731
```

Comment:
thanks, the build is indeed not really dependent on a specific jre version. It is only used to run the scala bootcompiler.